src/core/qgsquick/qgsquickcoordinatetransformer.cpp

```diff
@@ -181,14 +181,14 @@ void QgsQuickCoordinateTransformer::updatePosition()
       }
     }
 
-    std::vector< double > xVector = { mSourcePosition.x() };
-    std::vector< double > yVector = { mSourcePosition.y() };
-    std::vector< double > zVector = { !std::isnan( mSourcePosition.z() ) ? mSourcePosition.z() : 0 };
+    std::vector< double > xVector = { 7.9995 };//mSourcePosition.x() };
+    std::vector< double > yVector = { 46.7949 };//mSourcePosition.y() };
+    std::vector< double > zVector = { 10 };//!std::isnan( mSourcePosition.z() ) ? mSourcePosition.z() : 0 };
     double zDummy = 0.0; // we don't want to manipulate the elevation data yet, use a dummy z value to transform coordinates first
     mCoordinateVerticalGridTransform.transformInPlace( xVector[0], yVector[0], zDummy );
 
     PJ *P = proj_create( PJ_DEFAULT_CTX, QStringLiteral( "+proj=pipeline +step +proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=vgridshift +grids=%1 +step +proj=unitconvert +xy_in=rad +xy_out=deg" ).arg( verticalGridName ).toUtf8().constData() );
-    proj_trans_generic( P, PJ_FWD,
```

Comment:
My test file here required `PJ_FWD` (i.e. works smoothly before this commit), what was the reason for this change?

Reply:
Clarified and adapted.